sdk/appconfiguration/Azure.Data.AppConfiguration/api/Azure.Data.AppConfiguration.netstandard2.0.cs

```diff
@@ -9,19 +9,26 @@ public partial class ConfigurationClient
         public ConfigurationClient(System.Uri endpoint, Azure.Core.TokenCredential credential, Azure.Data.AppConfiguration.ConfigurationClientOptions options) { }
         public virtual Azure.Response<Azure.Data.AppConfiguration.ConfigurationSetting> AddConfigurationSetting(Azure.Data.AppConfiguration.ConfigurationSetting setting, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response<Azure.Data.AppConfiguration.ConfigurationSetting> AddConfigurationSetting(string key, string value, string label = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        [System.Diagnostics.DebuggerStepThroughAttribute]
```

Comment:
What are these?  We shouldn't be adding attributes to methods on the public API.

Reply:
This is auto generated when i ran the command.

Reply:
Verified the public api changes with command line. However, it still generates the debugger attributes as below but as no api changes so reverting the file to previous state. 

![image](https://user-images.githubusercontent.com/34852023/68945861-6a3d8080-07d7-11ea-8e19-ef509759a613.png)

Reply:
@pakrym do you know why that would be happening?

Reply:
It's an attribute automatically added by a compiler to methods marked as `async`. We don't count it as a breaking change.